hapi-fhir-jpaserver-test-utilities/src/main/java/ca/uhn/fhir/jpa/test/Batch2JobHelper.java

```diff
@@ -99,7 +99,7 @@ public JobInstance awaitJobHasStatus(String theBatchJobId, int theSecondsToWait,
 				.map(t -> t.getJobDefinitionId() + "/" + t.getStatus().name())
 				.collect(Collectors.joining("\n"));
 			String currentStatus = myJobCoordinator.getInstance(theBatchJobId).getStatus().name();
-			fail("Job still has status " + currentStatus + " - All statuses:\n" + statuses);
+			fail("Job still " + theBatchJobId + " has status " + currentStatus + " - All statuses:\n" + statuses);
```

Comment:
I think it should be like this:
```suggestion
			fail("Job " + theBatchJobId + " still has status " + currentStatus + " - All statuses:\n" + statuses);
```

Reply:
Thanks.